components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,39 @@
+import { Portal } from "@reach/portal";
```

Comment:
a `Portal` is a component that is appended to the end of the document body. This lets us make sure that it sits above everything else without needing to engage in z index wars. The `Panel` works in the same way.